backend/app/infrastructure/external_api/bitquery/blocks_reward.py

```diff
@@ -10,6 +11,7 @@
 )
 
 
+@cache.memoize(timeout=60)
```

Comment:
I've checked our usage and it's pretty low (limit is 10000, maybe we use different account. @mike-code can confirm, because he mentioned that we're a lot closer to it)
<img width="888" alt="Screenshot 2024-06-27 at 20 16 51" src="https://github.com/golemfoundation/octant/assets/25032165/f880259f-488a-4402-8231-527897427904">
Therefore, If this is the account, then we probably don't need to set cache anyway. If not - I think the time should be bigger then 60s, but I don't know what exactly. 1 hour maybe?

Reply:
@mike-code any opinion?

Reply:
I can't comment on how long the cache may be as I don't know the implications.

I'm not aware of a different account. There's no problem with us having a paid account, if needed.

Reply:
There is no real problem with keeping this cache for a longer time. Even from a day to seven days (time required for an org to implement a fix).

We shouldn't cache anything about recent blocks (younger than 20 minutes) though, since they might get reorganized and rewards values might change.